.github/workflows/pull_request.yaml

```diff
@@ -56,8 +56,9 @@ jobs:
           python-version: ${{ matrix.python }}
       - name: Install tox
         run: |
+          sudo apt-get -yy install ansible
```

Comment:
Why install it from apt rather than from pip? We are still reliant on pip to install the desired version.

Reply:
The strangest thing: Installing ansible with pip breaks molecule in the tox venv, presumably by leaving incompatible versions of files in common directories. System (ubuntu packaged) ansible doesn't.

Reply:
Wow, yeah. Please add a comment why we're doing that though. So it isn't accidentally code cleaned up.